security_monkey/views/item.py

```diff
@@ -247,6 +248,7 @@ def get(self):
         if 'active' in args:
             active = args['active'].lower() == "true"
             query = query.filter(ItemRevision.active == active)
+            join_accounts = True
```

Comment:
This didn't previously have `query = query.join((Account, Account.id == Item.account_id))`. Is this supposed to be here also?

Reply:
Yes. The `active` filter not working properly is what cause me to notice this. I then fixed the other bugs I saw due to the same root issue.